.github/workflows/linux.yml

```diff
@@ -22,3 +22,72 @@ jobs:
         with:
           name: ooniprobe-linux-amd64
           path: ./CLI/linux/amd64/ooniprobe
+
+  armhf_build:
+    runs-on: "ubuntu-latest"
+    name: Build on armhf Buster
+    steps:
+      - uses: actions/checkout@v2.1.0
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Build ooniprobe
+        id: runcmd
+        with:
+          arch: armv7
+          distro: buster
+          # distro: ubuntu20.04 # no linux-headers-arm64
+
+          # Not required, but speeds up builds by storing container images in
+          # a GitHub package registry.
+          githubToken: ${{ github.token }}
+
+          run: |
+            uname -a
+            apt-get -qq update
+            apt-get install -qq -y --no-install-recommends gcc git linux-headers-armmp wget musl-dev ca-certificates libc6-dev
+            wget --no-verbose https://golang.org/dl/go1.16rc1.linux-armv6l.tar.gz
+            tar xfz go1.16rc1.linux-armv6l.tar.gz
+            export PATH=$PATH:$(pwd)/go/bin
+            go version
+            go env
+            go build -tags netgo -ldflags='-s -w -extldflags -static' ./cmd/ooniprobe
```

Comment:
If you don't want to switch to Alpine, this is fine but, then, please omit `-static`. Statically linking with `glibc` is not a good idea because `glibc` insists on loading some modules dynamically. So, this is dangerous from the point of view of portability and can cause unexpected issues when running on different systems.